### PR TITLE
Align leadership state naming with implementation

### PR DESCRIPTION
**Pull request by achamayou**
A small change, extracted from #5769, but useful for trace validation with the realistic startup.

### CODE REVIEW

File: tla/consensus/ccfraft.tla [hunk 1]
```diff
@@ -1356,7 +1354,8 @@ StateTransitionsProp ==
 
 PendingBecomesFollowerProp ==
     \* A pending node that becomes part of any configuration immediately transitions to Follower.
-    [][\A s \in { s \in Servers : state[s] = Pending } : 
+    \* TODO: this is not right, the transition only takes place once the node receives a message
```
Comment (heidihoward) [inline tla/consensus/ccfraft.tla:1357]: I think this is fine as the `configurations` still only include the node after it has received at AE with its own addition

**Review by heidihoward — APPROVED** [review]

File: tla/consensus/ccfraft.tla (hunk 1 above)
Comment (achamayou) [reply tla/consensus/ccfraft.tla:1357]: Ok, yes, agreed. I will rephrase the "become", I think it's "becomes aware that it is part of any configuration".

File: tla/consensus/ccfraft.tla [hunk 2]
```diff
@@ -1356,7 +1354,8 @@ StateTransitionsProp ==
 
 PendingBecomesFollowerProp ==
     \* A pending node that becomes part of any configuration immediately transitions to Follower.
```
Comment (achamayou) [inline tla/consensus/ccfraft.tla:1356]:
```suggestion
    \* A pending node that becomes aware it is part of a configuration immediately transitions to Follower.
```

File: tla/consensus/ccfraft.tla (hunk 1 above)
Comment (achamayou) [inline tla/consensus/ccfraft.tla:1357]:
```suggestion
```

**Pull request merged**